src/demo/Box.java

```diff
@@ -0,0 +1,35 @@
+package demo;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class Box<T extends Fruit> {
+    private List<T> fruits = new ArrayList<>();
+
+    public void add(T fruit) {
+        fruits.add(fruit);
+    }
+
+    public void add(T... fruits) {
+        this.fruits.addAll(Arrays.asList(fruits));
```

Comment:
You should put the fruits of the same type in the box, so if you have the box with apples, you can't add a list of oranges there